CRM/Contact/Page/AJAX.php

```diff
@@ -786,29 +786,29 @@ public static function getDedupes() {
 
     $count = 0;
     foreach ($dupePairs as $key => $pairInfo) {
-      $pair =& $pairInfo['data'];
+      $pair = $pairInfo['data'];
```

Comment:
I'm inclined to leave it - just because some of those lines are quite long & it keeps us from going onto 2 lines with them